Cloudshell tutorial

josephlewis42 opened the pull request:
I've added a basic cloudshell tutorial about how to get going. Should fix #6 and provide a good source of info for people needing to debug.

josephlewis42 committed:
initial work on a tutorial

Change-Id: I383036a310bada3e36352fd775b0cbce27734033

josephlewis42 committed:
fixed half-completed sentence

Change-Id: Id18c0322d25ef16123b4bb49a2137e0d02900d45

rosbo commented on tutorial/config.yml:
```diff
@@ -0,0 +1,26 @@
+
+output.console:
+  pretty: true
+
```
nit: rm extra line

rosbo commented on tutorial/tutorial.md:
```diff
@@ -0,0 +1,101 @@
+
+Execute `gcsbeat` to fetch the
+
+	./gcsbeat -e -v -c tutorial/config.conf
```
Consider adding a line about what the user should expect to see.

rosbo commented on tutorial/config.yml:
```diff
@@ -0,0 +1,26 @@
+
+  delete: false
+  file_matches: "*.log"
+  file_exclude: "bak_*"
```
your bak filename in the demo has a dash `-` not an underscore `_`.

rosbo commented:
I like the CloudShell link. Super convenient.

josephlewis42 committed:
fixed config issues

bak_ to bak- so it'll match the file in the tutorial and removed extra trailing lines.

josephlewis42 committed:
added more context around what commands should do

josephlewis42 commented:
@rosbo Alright, I think I cleaned it all up. That's what I get for editing in my VDE and the CloudShell simultaneously :-)
